website/src/components/PreviewModeBar.js

```diff
@@ -1,17 +1,21 @@
 import cx from 'classnames';
 import { WORDPRESS_DOMAIN } from 'lib/constants';
 import { useState, useEffect } from 'react';
+import { BsInfoCircle } from 'react-icons/bs';
 import styles from './PreviewModeBar.module.scss';
 
 const WORDPRESS_EDIT_URL =
   process.env.WORDPRESS_EDIT_URL ||
   `https://${WORDPRESS_DOMAIN}/wp-admin/post.php?action=edit`;
```

Comment:
can use `WORDPRESS_URL` here instead (and update line 2) and remove `https://` and then this link will work for local dev/preview/prod

Reply:
Good call.